main/src/core/MonoDevelop.Ide/ExtensionModel/MainMenu.addin.xml

```diff
@@ -241,7 +241,7 @@
 	</ItemSet> 
 
 	<ItemSet id = "Help" _label = "_Help">
-		<CommandItem id = "MonoDevelop.Ide.Commands.HelpCommands.Help" />
+<!--		<CommandItem id = "MonoDevelop.Ide.Commands.HelpCommands.Help" /> -->
```

Comment:
Yes, it is quite confusing due to poor naming. "HelpCommands.Help" command has "API Documention" as the label for the menu item and it launches the DocBrowser.